dispatcher.py

```diff
@@ -78,10 +78,14 @@ def __init__(self, task_groups, max_workers_cnt, randomize):
         self.worker_next_id = 1
 
         tasks_cnt = 0
+        self.current_task_queue = SimpleQueue()
```

Comment:
Why the new queue is needed?

Reply:
Sergei (@sergw) helps me to remember the context: workers are separate processes, so we need a queue to pass information about a test that a worker receive.

I propose the following approach. A worker sends an object of specific type to the results queue (see 'Worker results' at worker.py:113) when receive a task. The HangWatcher listener (listeners.py) checks for this type of an object in `process_result` and saves necessary info. Then it can be used in `process_timeout` method.

So, we’ll avoid usage of extra queue and we’ll reuse our flexible approach to handle events depending of its type.